tests/python-gpu/test_gpu_prediction.py

```diff
@@ -185,6 +190,20 @@ def predict_dense(x):
         base_margin = cp_rng.randn(rows)
         self.run_inplace_base_margin(booster, dtrain, X, base_margin)
 
+        # Create a wide dataset
+        X = cp_rng.randn(100, 10000)
+        y = cp_rng.randn(100)
+
+        missing_idx = [i for i in range(0, X.shape[1], 16)]
+        X[:, missing_idx] = missing
+        reg = xgb.XGBRegressor(tree_method="gpu_hist", n_estimators=8, missing=missing)
+        reg.fit(X, y)
+
+        gpu_predt = reg.predict(X)
+        reg.set_params(predictor="cpu_predictor")
+        cpu_predt = reg.predict(X)
```

Comment:
This is not inplace as we have specified `cpu_predictor`.  (When predictor is specified we have to fallback to normal predict as we don't know what are the inputs).